schema-registry/src/main/scala/net.manub.embeddedkafka/schemaregistry/embeddedKafkaWithSchemaRegistry.scala

```diff
@@ -0,0 +1,203 @@
+package net.manub.embeddedkafka.schemaregistry
+
+import java.util.Properties
+
+import io.confluent.kafka.schemaregistry.RestApp
+import io.confluent.kafka.schemaregistry.avro.AvroCompatibilityLevel
+import kafka.server.KafkaServer
+import net.manub.embeddedkafka.EmbeddedKafka.servers
+import net.manub.embeddedkafka.{
+  EmbeddedKafka,
+  EmbeddedKafkaConfig,
+  EmbeddedKafkaSupport,
+  EmbeddedServer
+}
+import org.apache.kafka.clients.consumer.{ConsumerConfig, OffsetResetStrategy}
+import org.apache.kafka.clients.producer.ProducerConfig
+
+trait EmbeddedKafkaWithSchemaRegistry
```

Comment:
Not sure. Schema Registry is a plus and not everybody uses it. I can see your point but I also think it may lead to confusion.  Would be great to see other people feedback.

Reply:
Expliciteness through the class name is better than in the package IMO. We're talking user experience here

Reply:
I'm not arguing it doesn't exist, not it's right or wrong. As a user doing integration tests I'd rather have explicit names, that's all.

Reply:
@manub will have the last word, of course, but since many classes already have very long names, I was trying not to make them even longer, kinda like Objective-C...